Resources/Private/JavaScript/Host/Redux/Sagas/UI/Inspector/index.js

```diff
@@ -0,0 +1,29 @@
+import {takeEvery} from 'redux-saga';
+import {put} from 'redux-saga/effects';
+import {$get} from 'plow-js';
+
+import {actionTypes, actions} from 'Host/Redux/';
+
+const getTransientInspectorValues = $get(['ui', 'inspector', 'valuesByNodePath']);
+
+export function* applyInspectorState(getState) {
+    yield* takeEvery(actionTypes.UI.Inspector.APPLY, function* applyAllChanges(action) {
+        const state = getState();
+        const {nodeContextPath} = action.payload;
+
+        const transientNodeInspectorValues = $get([nodeContextPath], getTransientInspectorValues(state));
+
+        for (const key of Object.keys(transientNodeInspectorValues)) {
+            yield put(actions.Changes.add({
```

Comment:
Maybe we can gather all changes into an array here, and add persist them in one go? That would be a huge performance win.